docs/changelog.md

```diff
@@ -1,3 +1,26 @@
+## v2.5.0next [20xx-xx-xx]
+
+### Added
+- [FOGL-8942]: The ordering of the tabs in configuration user interface is no longer alphabetical, but controlled by the configuration category itself.[#471](https://github.com/fledge-iot/fledge-gui/pull/471)
+- [FOGL-8990]: The GUI rendering of lists in the configuration options has been improved.[#462](https://github.com/fledge-iot/fledge-gui/pull/462)
+- [FOGL-8841]: The user interface now hides configuration options the user does not have permission to change. [#437](https://github.com/fledge-iot/fledge-gui/pull/437) 
+
+### Fixed
+- [FOGL-8588]: Filter installation.[#467](https://github.com/fledge-iot/fledge-gui/pull/467)
+- [FOGL-9129]: Allow to choose a filter plugin, when clicked on + icon of filter node.[#466](https://github.com/fledge-iot/fledge-gui/pull/466)
```

Comment:
This should go under:

Visual programming based Flow Editor interface improvements ?